syft/frameworks/torch/tensors/interpreters/native.py

```diff
@@ -294,13 +294,21 @@ def handle_func_command(cls, command):
 
             # build the new command
             new_command = (cmd, None, new_args, new_kwargs)
+            # Check that the function is not overloaded - for syft tensors
+            try:
+                # Try to get recursively the attributes in cmd = "<attr1>.<attr2>.<attr3>..."
+                command = cls.rgetattr(cls, new_command)
```

Comment:
The 2nd arg of `rgetattr` should be a string with a structure like so: `attr1.attr2.attr3...`
But here you are feeding the function with `new_command` which is a tuple.

Shouldn't you be feeding it with cmd instead?

Reply:
That's correct. Thanks for bringing this to notice :).
I have replaced ```new_command``` with ```cmd```. It was raising `AttributeError` everytime. 
Now it's not working for one case though when we pass **wrapper tensor as shifts argument**. It's raising the exception `RuntimeError: Error, Please consider calling ".get" method instead of ".item" method, so you can be safely getting the item you need.`